addOns/oast/src/main/java/org/zaproxy/addon/oast/services/boast/BoastOptionsPanelTab.java

```diff
@@ -135,6 +138,8 @@ private void registerButtonAction() {
             boastService.register();
             getBoastServersTableModel().fireTableDataChanged();
         } catch (Exception exception) {
+            exception.printStackTrace();
```

Comment:
You might want to put a TODO on this so that it's removed when this is no longer WIP.